pkg/builder.go

```diff
@@ -28,7 +27,7 @@ const (
 // Git.
 //
 // Run returns on of the Status* status code constants.
-func Run(sshHostIP string, sshHostPort int, cmd string) int {
+func RunBuilder(sshHostIP string, sshHostPort int, sshServerCircuit *sshd.Circuit) int {
```

Comment:
I removed `cmd` and changed the name to add specificity to this function. Objections to leaving it in?

For context, `cmd`'s purpose was to tell `Run` what cookoo route to run, but none were supported except for `"boot"` (which corresponds to running the builder SSH server).